4. loadSpaceCrew method added, some fixes

vilzerd opened the pull request.

vilzerd committed:
5. loadSpaceCrew method added, some fixes

vilzerd committed:
5. loadSpaceCrew method added, some fixes

Jakub112358 commented on src/main/java/iss/DAO/ISSLocationDAO.java:
```diff
@@ -7,37 +7,33 @@
         Transaction transaction = session.beginTransaction();
-        List<ISSLocation> issLocationList = session.createQuery("SELECT * FROM craft_info", ISSLocation.class)
+        List<ISSLocation> issLocationList = session
+                .createQuery("SELECT * FROM iss_location", ISSLocation.class)
```
In method .createQuery, query should be given in HQL, not SQL. There is a method .createSQLQuery for SQL queries.

Jakub112358 committed:
Update ISSLocationDAO.java

fixed constructor method
fixed query (hql instead of sql)

Jakub112358 committed:
Update ISSLocationDAO.java

Jakub112358 committed:
Update SpaceCrewDAO.java

fixed loadSpaceCrew method

Jakub112358 commented on src/main/java/iss/controller/Controller.java:
```diff
@@ -58,13 +58,17 @@ public class Controller {
+            speedLabel.setText(service.getSpeed());
+        } catch (InterruptedException e) {
+            throw new RuntimeException(e);
+        }
```
this part will be removed from here anyway

Jakub112358 committed:
Update ISSLocation.java

timestamp should be int or maybe long (int should work until 2035 or something like that), but for sure not double

Jakub112358 committed:
Update ISSVelocity.java

int is optimal variable for velocity - we don't need more accuracy than provided by int due to HUGE inacurracy in time measurment and moreover int will be easier to use in the rest of the program

Jakub112358 reviewed (approved):
accepted with few fixes ;)
